STX.REST.RESTFulSense.Clients/Models/Services/ErrorMappers/StatusDetail.cs

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
 // ----------------------------------------------------------------------------------
 
-namespace STX.REST.RESTFulSense.Clients.Models.Errors
+namespace STX.REST.RESTFulSense.Clients.Models.Services.ErrorMappers
 {
     internal class StatusDetail
     {
```

Comment:
I think this properties should be all of them init instead set

Reply:
Should it be handled in a separate issue as requested for the upgrade. It is DATA not FOUNDATIONS.

Reply:
yeah, your are right, I noticed because of the merge with the main,